bin/watch.ts

```diff
@@ -20,26 +22,27 @@ function* start(): Sequence {
   });
 
   let current = { halt: (x = undefined) => x };
+  let context = yield self;
   let restart = () => {
     current.halt();
-    current = this.fork(function*() {
+    current = context.spawn(fork(function*() {
```

Comment:
In order to imperatively invoke an operation on a context, use the `spawn` method. Note that `spawn` does not do anything except add the child to the child list, so without wrapping it in a `fork` or a `monitor` higher-order operation, it won't actually do anything with the return value. We could probably work on the `spawn` API going forward because it's really easy to shoot yourself in the foot with it.